#! /usr/bin/env python
PACKAGE = "color_target_tracker"
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lower_hue", int_t, 0, "Lower hue", 0, 0, 179)
gen.add("upper_hue", int_t, 0, "Upper hue", 179, 0, 179)
gen.add("lower_saturation", int_t, 0, "Lower saturation", 100, 0, 255)
gen.add("upper_saturation", int_t, 0, "Upper saturation", 255, 0, 255)
gen.add("lower_value", int_t, 0, "Lower value", 100, 0, 255)
gen.add("upper_value", int_t, 0, "Upper value", 255, 0, 255)

exit(gen.generate(PACKAGE, "color_target_tracker", "ColorRange"))
